specs/lifespan.rst

```diff
@@ -0,0 +1,55 @@
+=================
+Lifespan Protocol
+=================
+
+**Version**: 2.1 (2018-07-01)
+
+The Lifespan ASGI sub-specification outlines how to communicate
+lifespan events such as startup and cleanup within ASGI. The lifespan
+being refered to is that of main event loop.
+
+
+Scope
+'''''
+
+The lifespan scope exists for the duration of the event loop. The
+scope itself contains basic metadata,
+
+* ``type``: ``lifespan``
+
+* ``asgi``: A dictionary containing a ``version`` key, which itself is
```

Comment:
Don't think this needs to be redefined in every protocol.

Reply:
Changed.